src/app/app.component.spec.ts

```diff
@@ -117,27 +120,32 @@ describe('AppComponent', () => {
 
     // expect(component instanceof AppComponent).to.equal(true);
     expect(component).to.be.an.instanceOf(AppComponent);
-    expect(component.menuItems.length).to.equal(2);
+    expect(component.menuItems.length).to.equal(3);
 
     expect(component).to.haveOwnProperty('activeItem');
     expect(component.activeItem).to.be.an.instanceOf(MenuItem);
     // TODO : Check that the rootPage is the same as the component of the activeItem
     // expect(component.rootPage).to.eql(component.activeItem.component);
 
     // Some initialization should not be done until platform is ready
-    expect(splashScreenMock.hide.called, 'splashScreen.hide called').to.equal(false);
-    expect(statusBarMock.styleDefault.called, 'statusBar.styleDefault called').to.equal(false);
+    expect(splashScreenMock.hide.called, 'splashScreen.hide() called').to.equal(false);
+    expect(statusBarMock.styleDefault.called, 'statusBar.styleDefault() called').to.equal(false);
 
-    expect(asSpy(moment.locale).args, 'moment.locale called').to.eql([ [ 'fr' ] ]);
+    expect(asSpy(moment.locale).args, 'moment.locale() called').to.eql([ [ 'fr' ] ]);
 
-    expect(translateService.setDefaultLang.args, 'translateService.setDefaultLang called').to.eql([ [ 'fr' ] ]);
-    expect(translateService.setTranslation.args, 'translateService.setTranslation called').to.eql([ [ 'fr', fr ] ]);
-    expect(translateService.use.args, 'translateService.use called').to.eql([ [ 'fr' ] ]);
+    expect(translateService.setDefaultLang.args, 'translateService.setDefaultLang() called').to.eql([ [ 'fr' ] ]);
+    expect(translateService.setTranslation.args, 'translateService.setTranslation() called').to.eql([ [ 'fr', fr ] ]);
+    expect(translateService.use.args, 'translateService.use() called').to.eql([ [ 'fr' ] ]);
 
     fixture.detectChanges();
 
-    const title = fixture.debugElement.query(By.css('ion-title'));
-    expect(title.nativeElement.textContent).to.equal(fr.app);
+    // @Simon: that's very strange... with query(By.all()), I'd expect to get all elements of the page
+    // I kinda do, but I only get elements that are present in the `app.html` file,
+    // as if the page component's template was not included...
```

Comment:
Yeah, the use of `query(By.all())` was only done to check the content of the component's template.

But you're right, the template is overriden previously. I propose we just remove this expectation ; unless there's something I'm missing ?

Reply:
Since the title that was checked is no longer there, that sounds good to me.

We might want to see if you can use the same kind of stub you used elsewhere to override the map page.